.github/workflows/ci.yml

```
@@ -102,6 +102,31 @@ jobs:
           csharp/bin/Release/net8.0/win-x64/publish/hello.exe
           csharp/bin/Release/net8.0/win-x64/publish/info.txt
 
+  build_d:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dlang-community/setup-dlang@v1
+      with:
+        compiler: dmd-2.103.1
```

I leave everything I can at `latest` so that there's less work to update it.

```suggestion
        compiler: dmd-latest
```